### ISSUE TITLE
Too few images, skipping this tag

### ISSUE THREAD

**Issue by kharazian** [InstaPy/InstaPy#6646]
When search for posts of tag, InstaPy show this message "Too few images, skipping this tag".
Anybody has any idea.

**Comment by official-antique** [comment]
@kharazian It's happening because of a change to the xpath(s) of /explore/tag. Posts cannot be correctly found because of it.

I'm currently in the process of trying to get it to work.

**Comment by lagriffone** [comment]
@official-antique, Is it possible to get it work? Because seems like they completely removed the recent posts section from the Instagram web. I don't see it at all. Wonder if its a regional thing or if its removed for everybody?

**Comment by official-antique** [comment]
@lagriffone Do you have an endpoint for me to test? Not too sure what that is sorry.

**Comment by lagriffone** [comment]
@official-antique Yeah since yesterday the explore/tags page (ex: "instagram.com/explore/tags/catoftheday"), only has the **Top posts** section and the **Recent posts** section seems to be completely removed.

**Comment by official-antique** [comment]
@lagriffone I did see that last night, the script wouldn't find the main posts so that'll have to be removed most likely

**Comment by official-antique** [comment]
@lagriffone okay, just looked and Recent is in fact still there however, it's not shown to the user. I'll need to rewrite the code to use the hidden data.

**Comment by orirzn1** [comment]
Any updates?

**Comment by thegreatref** [comment]
I have the same issue
